amsterdam/Issues/taxizoneOntheffing/description.xml

```diff
@@ -0,0 +1,60 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo Taxi zone exemption Amsterdam</en>
+        <nl>Demo Taxizone-ontheffing Amsterdam</nl>
+    </Name>
+    <ShortName>
+        <en>Demo Taxi zone exemption Amsterdam</en>
+        <nl>Demo Taxizone-ontheffing Amsterdam</nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>amsterdam</IssuerID>
+    <CredentialID>taxizoneOntheffing</CredentialID>
+    <Description>
+        <en>Taxi zone exemption Amsterdam</en>
+        <nl>Taxizone-ontheffing Amsterdam</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
+    <IssueURL>
+        <en>https://ontheffingen-acc.amsterdam.nl/taxizone/</en>
+        <nl>https://ontheffingen-acc.amsterdam.nl/taxizone/</nl>
+    </IssueURL>
+
+    <RevocationServers>
+        <RevocationServer>https://ontheffingen-acc.amsterdam.nl/irma-api/irma</RevocationServer>
+    </RevocationServers>
+
+    <Attributes>
+        <Attribute revocation="true" />
+        <Attribute id="ontheffingsnummer">
+            <Name>
+                <en>Exemption number</en>
+                <nl>Ontheffingsnummer</nl>
+            </Name>
+            <Description>
+                <en>Exemption number</en>
+                <nl>Ontheffingsnummer</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="geldigVanaf" optional="true">
+            <Name>
+                <en>Exemption valid from</en>
+                <nl>Ontheffing geldig vanaf</nl>
+            </Name>
+            <Description>
+                <en>Your taxi zone exemption is valid from this date</en>
+                <nl>Je taxizone-ontheffing is geldig vanaf deze datum</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="geldigTotEnMet" optional="true">
```

Comment:
The attribute identifiers should be in English. That's our guideline in pbdf and it's best practice to keep it consistent here.

Reply:
Changed it, also the credentialType itself has been renamed to be in English.

Might be good to add this to the issuer documentation as well.